src/main/java/net/minestom/server/world/biomes/BiomeManager.java

```diff
@@ -19,8 +20,48 @@
 public final class BiomeManager {
     private final Map<Integer, Biome> biomes = new ConcurrentHashMap<>();
 
+    // https://minecraft.fandom.com/wiki/Rain
+    private final static Double SNOW_TEMPERATURE = 0.15;
+    private static final boolean loadBiomes = Boolean.getBoolean("minestom.load-vanilla-biomes");
+
     public BiomeManager() {
         addBiome(Biome.PLAINS);
+
+        if (loadBiomes) {
```

Comment:
Perhaps this would be better as a function on BiomeManager, eg `loadVanillaBiomes()` which you can call once during startup to load them